core-model/src/main/java/com/sequenceiq/cloudbreak/domain/view/RdsConfigWithoutCluster.java

```diff
@@ -0,0 +1,41 @@
+package com.sequenceiq.cloudbreak.domain.view;
+
+import com.sequenceiq.cloudbreak.api.endpoint.v4.common.DatabaseVendor;
+import com.sequenceiq.cloudbreak.api.endpoint.v4.common.ResourceStatus;
+import com.sequenceiq.cloudbreak.domain.RdsSslMode;
+import com.sequenceiq.cloudbreak.service.secret.domain.Secret;
+
+public interface RdsConfigWithoutCluster {
```

Comment:
Would it make sense to have `RDSConfig` actually implement this IF?

Reply:
Nope, it is a Spring data view. We are projecting into these fields the SQL record. However, I have never tested what happen if I add as an interface

Reply:
OK, let us leave it as is for now.